src/components/Popover/Popover.tsx

```diff
@@ -119,10 +80,12 @@ const PopoverGroup = (props: ExtractProps<typeof HeadlessPopover.Group>) => (
   <HeadlessPopover.Group {...props} />
 );
 
+// TODO-AH: consolidate renderprop types between all components
```

Comment:
There are a few items with this comment format to tidy up still, or i may just pull those into a separate QoL ticket